### PR TITLE
Adding README.md to TFLite evaluation/tasks

### PR DESCRIPTION
**Pull request by Harsh188**
PR related to #43294. 

Adding documentation under `evaluation/tasks` to guide users who are directed from the [TFLite documentation guide](https://www.tensorflow.org/lite/performance/post_training_quantization#model_accuracy).

### CODE REVIEW

**Comment by Harsh188** [comment]
@bhack any suggestions?

**Comment by Harsh188** [comment]
@gbaned requesting reviewers

File: tensorflow/lite/tools/evaluation/tasks/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# TFLite Model Task Evaluation
```
Comment (multiverse-tf) [inline tensorflow/lite/tools/evaluation/tasks/README.md:1]: Thanks for contributing this file. Recently, we have described the purpose of these tools and how they are used in our tf website (https://www.tensorflow.org/lite/performance/delegates#accuracy_correctness). So, could you adapt this short accordingly? Many thanks!

File: tensorflow/lite/tools/evaluation/tasks/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+### Inference Diff Tool
+This binary compares TensorFlow Lite execution in single-threaded CPU inference and user-defined inference.
+
+### Image Classification Evaluation
+This binary evaluates TensorFlow Lite models trained for the [ILSVRC 2012 image classification task.](http://www.image-net.org/challenges/LSVRC/2012/)
+
+### Object Detection Evaluation
+This binary evaluates TensorFlow Lite models trained for the bounding box-based [COCO Object Detection](https://cocodataset.org/#detection-eval) task.
+
+For a more detailed specification of each tool refer to:
```
Comment (multiverse-tf) [inline tensorflow/lite/tools/evaluation/tasks/README.md:17]: Consider merging each item to the sections above accordingly?

Again, I think it's also worth referring to "https://www.tensorflow.org/lite/performance/delegates#accuracy_correctness" at the end.

File: tensorflow/lite/tools/evaluation/tasks/README.md (hunk 1 above)
Comment (srjoglekar246) [reply tensorflow/lite/tools/evaluation/tasks/README.md:1]: +1. Thanks for doing this @Harsh188 !

Comment (Harsh188) [reply tensorflow/lite/tools/evaluation/tasks/README.md:1]: Thanks for the review!

**Comment by Harsh188** [comment]
Thanks for the review. I added snippets of information from [Tools for Evaluation](https://www.tensorflow.org/lite/performance/delegates#tools_for_evaluation). For the most part, I tried to keep it short and redirected people to the TFLite guide site to keep it consistent.

File: tensorflow/lite/tools/evaluation/tasks/README.md [hunk 3]
```diff
@@ -2,20 +2,19 @@
 
 This page describes how you can check the accuracy of quantized models to verify that any degradation in accuracy is within acceptable limits.
 
+## Accuracy & correctness
+TensorFlow Lite has two types of tooling to measure how accurately a delegate behaves for a given model: Task-Based and Task-Agnostic. 
+
+For more information visit the [Tools for Evaluation](https://www.tensorflow.org/lite/performance/delegates#tools_for_evaluation) page.
```
Comment (srjoglekar246) [inline tensorflow/lite/tools/evaluation/tasks/README.md]: Point to this instead, for the exact section: https://www.tensorflow.org/lite/performance/delegates#accuracy_correctness

**Review by srjoglekar246 — CHANGES_REQUESTED** [review]
Looks good to me after the minor link fix.

**Review by srjoglekar246 — APPROVED** [review]

**Pull request merged**